dataprofiler/labelers/base_data_labeler.py

```diff
@@ -716,10 +715,10 @@ def fit(self, x, y, validation_split=0.2, labels=None, reset_weights=False,
         if reset_weights:
             self._model.reset_weights()
 
-        # shuffle input data, these are pandas.DataFrame or pandas.Series
+        # shuffle input data
         shuffle_inds = np.random.permutation(num_samples)
-        x = x.loc[shuffle_inds].reset_index(drop=True)
-        y = y.loc[shuffle_inds].reset_index(drop=True)
+        x = x[shuffle_inds]
+        y = y[shuffle_inds]
```

Comment:
Why are we changing this?

Reply:
They are np arrays now, not dataframes, so we need to update use of .loc